### PR TITLE
Add manifest for GnuCash

### PR DESCRIPTION
**Pull request by Barthalion**
- [x] Add MySQL support to libdbi
- [x] Add PostgreSQL support to libdbi
- [x] Remove unneeded files from the app directory
- [x] Fix icon exporting

### CODE REVIEW

File: org.gnucash.GnuCash.json [hunk 1]
```diff
@@ -0,0 +1,168 @@
...
+  "runtime": "org.gnome.Platform",
+  "runtime-version": "3.24",
+  "sdk": "org.gnome.Sdk",
+  "command": "gnucash",
+  "rename-icon": "gnucash",
+  "rename-desktop-file": "gnucash.desktop",
+  "finish-args": [
+    "--share=ipc",
+    "--socket=x11",
+    "--socket=wayland",
```
Comment (TingPing) [inline org.gnucash.GnuCash.json]: Gtk2, no Wayland.

File: org.gnucash.GnuCash.json [hunk 2]
```diff
@@ -0,0 +1,168 @@
...
+    {
+      "name": "webkitgtk",
+      "config-opts": [
+        "--disable-webkit2",
+        "--with-gtk=2.0"
+      ],
+      "sources": [
+        {
+          "type": "archive",
+          "url": "https://webkitgtk.org/releases/webkitgtk-2.4.11.tar.xz",
```
Comment (TingPing) [inline org.gnucash.GnuCash.json]: Is this possibly an optional dep? I don't like the idea of adding new packages with CVEs that will never be fixed.

File: org.gnucash.GnuCash.json [hunk 3]
```diff
@@ -0,0 +1,168 @@
...
+          "sha256": "165070beb67b84580afe80a8a100b674a81d553ab791acd72ac0c655f4fadb15"
+        },
+        {
+          "type": "patch",
+          "path": "goffice-0.8-use-pkgconfig-for-pcre.patch"
+        },
+        {
+          "type": "shell",
+          "commands": [
+            "rm -f configure"
```
Comment (TingPing) [inline org.gnucash.GnuCash.json]: `"rm-configure": true`

File: org.gnucash.GnuCash.json (hunk 2 above)
Comment (Barthalion) [inline org.gnucash.GnuCash.json]: That's the exact point why I'm working on this – it's not going to stay in repos forever, and there is long way ahead before upstream moves to supported libraries.

File: org.gnucash.GnuCash.json (hunk 3 above)
Comment (Barthalion) [inline org.gnucash.GnuCash.json]: Right. Will fix it.

File: org.gnucash.GnuCash.json (hunk 2 above)
Comment (Barthalion) [inline org.gnucash.GnuCash.json]: Also without network access it's still better than nothing.

Comment (TingPing) [inline org.gnucash.GnuCash.json]: Good point, I suppose it is acceptable then.

**Comment by Barthalion** [comment]
Obviously MySQL and PostgreSQL support requires network access but I guess it should stay disabled as user can override it anyway.

@TingPing: do you see anything else that could be improved?

**Comment by Barthalion** [comment]
Merged.

**Pull request closed without merging**